### PR TITLE
Allow selecting custom rpc from menu even when custom rpc is selected

### LINKED ISSUE #541: Cant create new alt network when already connected to alt network
image

### PR DESCRIPTION
**Pull request by danfinlay**
Fixes #541 

Now when a custom network is selected, it is shown as selected in the drop-down menu, instead of just selecting `Custom RPC`, which made it impossible to select a new custom RPC from the menu when one was already selected.

screen shot 2016-08-29 at 12 35 52 pm

### CODE REVIEW

File: development/states/custom-rpc.json [hunk 1]
```diff
@@ -158,7 +158,7 @@
       }
     ],
     "selectedAddress": "0xfdea65c8e26263f6d9a1b5de9555d2931a33b825",
-    "network": "loading",
+    "network": "166",
```
Comment (kumavis) [inline development/states/custom-rpc.json]: @flyswatter whats the significance of this, if any?

Comment (flyswatter) [inline development/states/custom-rpc.json]: I put it in its own commit so it has a note in the git history.

This state file should represent a custom RPC being selected, but this attribute had the state also representing a loading state.

I set it to a hard network ID so it doesn't show loading.

I came up with 166 because at the time I thought that was the consensysnet ID, but actually it's 161.
